docs/rules/no-debug.md

```diff
@@ -28,6 +30,12 @@ const { screen } = require('@testing-library/react');
 screen.debug();
 ```
 
+If you want to allow the use of some debugging functions, you can configure what names this rule checks for with the `utilsToCheckFor` option:
```

Comment:
Could you include here the full list of debugging utils that can be switched on/off, please? Basically those from `DEBUG_UTILS`.

Reply:
also it would be good if I can write valid json 🤦